### PR TITLE
EZP-32199 - SF documentation change

### PR DESCRIPTION
**Pull request by dabrt**
| Question      | Answer |
| ------------- | --- |
| JIRA Ticket   | EZP-32199 |
| Versions      | master |

Add a note about SiteAccess matching priority.

### CODE REVIEW

File: docs/site_organization/site_factory.md [hunk 1]
```diff
@@ -36,6 +36,13 @@
     
     If all required fields are filled out you can select **Create,** and the site will be added to the site list in the **Site** tab.
     
+    !!! note
+
+    A SiteAccess that you create in SiteFactory is always treated with lower priority than a SiteAccess
```
Comment (DominikaK) [inline docs/site_organization/site_factory.md:41]: Indent missing in the whole note

Comment (DominikaK) [inline docs/site_organization/site_factory.md:41]:
```suggestion
    A SiteAccess that you create in Site Factory is always treated with lower priority than a SiteAccess 
```

File: docs/site_organization/site_factory.md [hunk 2]
```diff
@@ -36,6 +36,13 @@
     
     If all required fields are filled out you can select **Create,** and the site will be added to the site list in the **Site** tab.
     
+    !!! note
+
+    A SiteAccess that you create in SiteFactory is always treated with lower priority than a SiteAccess 
+    defined by the administrator as part of the [configuration procedure](https://doc.ibexa.co/en/latest/guide/siteaccess/#configuring-siteaccesses).
```
Comment (DominikaK) [inline docs/site_organization/site_factory.md:42]:
```suggestion
    defined by the administrator as part of [configuration](https://doc.ibexa.co/en/latest/guide/siteaccess/#configuring-siteaccesses). 
```

File: docs/site_organization/site_factory.md [hunk 3]
```diff
@@ -36,6 +36,13 @@
     
     If all required fields are filled out you can select **Create,** and the site will be added to the site list in the **Site** tab.
     
+    !!! note
+
+    A SiteAccess that you create in SiteFactory is always treated with lower priority than a SiteAccess 
+    defined by the administrator as part of the [configuration procedure](https://doc.ibexa.co/en/latest/guide/siteaccess/#configuring-siteaccesses). 
+    For example, if you create a site that uses the `fr` path in SiteFactory, and the administrator
```
Comment (DominikaK) [inline docs/site_organization/site_factory.md:43]:
```suggestion
    For example, if you create a site that uses the `fr` path in Site Factory, and the administrator 
```

**Review by DominikaK — CHANGES_REQUESTED** [review]

**Review by DominikaK — APPROVED** [review]

**Review by mikadamczyk — APPROVED** [review]

**Pull request merged**